docs/features/webhooks.md

```
@@ -4,32 +4,37 @@ Webhooks offer a powerful mechanism to receive real-time notifications of events
+    - `messageId`: Unique ID  
+    - `phoneNumber`: Recipient  
+    - `simNumber`: SIM index (nullable)  
+    - `sentAt`: Local timestamp
+
+- :white_check_mark: **sms:delivered**
+    - `messageId`: Unique ID  
+    - `phoneNumber`: Recipient  
+    - `deliveredAt`: Local timestamp
+
+- :x: **sms:failed**
+    - `messageId`: Unique ID  
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)